src/features/game/expansion/components/ChineseNewYearQuest.tsx

```diff
@@ -0,0 +1,222 @@
+      </p>
+    </div>
+  );
+};
+
+export const ChineseNewYearQuest: React.FC = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  const bumpkin: Partial<Equipped> = {
+    body: "Beige Farmer Potion",
+    hair: "Basic Hair",
+    pants: "Blue Suspenders",
+    shirt: "Blue Farmer Shirt",
+    tool: "Sword",
+    shoes: "Black Farmer Boots",
+    background: "Farm Background",
+    hat: "Lion Dance Mask",
+  };
+
+  return (
+    <>
+      <img
+        src={island}
+        className="absolute"
+        style={{
+          width: `${PIXEL_SCALE * 226}px`,
+          bottom: `${GRID_WIDTH_PX * 10}px`,
+          left: `${GRID_WIDTH_PX * -21}px`,
+        }}
+      />
+      <img
+        src={incenseLeft}
+        className="absolute z-20"
+        style={{
+          width: `${PIXEL_SCALE * 15}px`,
+          bottom: `${GRID_WIDTH_PX * 16.87}px`,
+          left: `${GRID_WIDTH_PX * -16.43}px`,
+        }}
+      />
+
+      <img
+        src={incenseRight}
+        className="absolute z-20"
+        style={{
+          width: `${PIXEL_SCALE * 15}px`,
+          bottom: `${GRID_WIDTH_PX * 16.87}px`,
+          left: `${GRID_WIDTH_PX * -12.5}px`,
+        }}
+      />
+
+      <img
+        src={lampNpc}
+        className="absolute z-20"
+        style={{
+          width: `${PIXEL_SCALE * 22}px`,
+          bottom: `${GRID_WIDTH_PX * 13.87}px`,
+          left: `${GRID_WIDTH_PX * -9.12}px`,
+        }}
+      />
+
+      <div
+        className="absolute z-20"
+        style={{
+          top: `${GRID_WIDTH_PX * 23}px`,
+          left: `${GRID_WIDTH_PX * -17}px`,
+        }}
+      >
+        <img
+          src={shadow}
+          className="absolute z-10"
+          style={{
+            width: `${PIXEL_SCALE * 16}px`,
+            bottom: `${PIXEL_SCALE * 4}px`,
+            left: `${PIXEL_SCALE * 2}px`,
+          }}
+        />
+        <img
+          src={mrChu}
+          className="relative left-0 top-0 z-20"
+          style={{
+            width: `${PIXEL_SCALE * 21}px`,
+          }}
+        />
+      </div>
+      <div
+        className="absolute z-20"
+        style={{
+          top: `${GRID_WIDTH_PX * 24}px`,
+          left: `${GRID_WIDTH_PX * -12}px`,
+        }}
+      >
+        <img
+          src={shadow}
+          className="absolute z-10"
+          style={{
+            width: `${PIXEL_SCALE * 15}px`,
+            bottom: `${PIXEL_SCALE * -2}px`,
+          }}
+        />
+        <img
+          src={npc}
+          className="relative left-0 top-0 cursor-pointer hover:img-highlight z-20"
+          onClick={() => setShowModal(true)}
+          style={{
+            width: `${PIXEL_SCALE * 16}px`,
+          }}
+        />
+      </div>
+
+      <Modal show={showModal} onHide={() => setShowModal(false)} centered>
+        {!todaysFind ? (
+          <CloseButtonPanel
+            title="The New Year is coming!"
+            bumpkinParts={bumpkin}
+          >
+            {ModalEventComing()}
```

Comment:
What will happen on the 27th? @Rofrtd 
 
Does this modal show again until it is removed from the code?